add how to run a sequencer docs (#2229)
OffchainLabs/arbitrum-docs: arbitrum-docs/run-arbitrum-node/sequencer/04-run-sequencer-node.mdx

Please note that:

- These minimum requirements for RAM and CPU are recommended for nodes that process a small amount of RPC requests. For nodes that require processing multiple simultaneous requests, both RAM and number of CPU cores will need to be scaled with the amount of traffic being served.
- Single core performance is important. If the node is falling behind and a single core is 100% busy, it is recommended to update to a faster processor
- The minimum storage requirements will change over time as the chain grows. Using more than the minimum requirements to run a robust full node is recommended.

## Recommended Nitro version

:::caution

Even though there are alpha and beta versions of the Arbitrum Nitro software, only release versions should be used when running your node. Running alpha or beta versions is not supported and might lead to unexpected behaviors.

:::

Latest Docker image: <code>@@latestNitroNodeImage=offchainlabs/nitro-node:v3.5.5-90ee45c@@</code>

## Required parameters

### 1. Sequencer node parameters

The following parameters are required to run a sequencer node:

#### 1. Enable sequencer

Enable the sequencer mode:

```shell
--node.sequencer=true
```

> #### 2. Make the node acts and posts to l1 as a sequencer

Reviewer: ```suggestion
#### 2. Make the node act as a sequencer and post to L1
```

Enable the sequencer execution:

```shell
--execution.sequencer.enable=true
--execution.sequencer.max-tx-data-size=85000
--execution.sequencer.max-block-speed=250ms
```

#### 3. Enable delayed sequencer

> Enable your node to read and include transaction from parent chain delayed inbox.

Reviewer: ```suggestion
Enable your node to read and include transactions from the parent chain delayed inbox.
```

```shell
--node.delayed-sequencer.enable=true
--node.delayed-sequencer.use-merge-finality=false
--node.delayed-sequencer.finalize-distance=1
```

#### 4. Enable batch poster

> Enable your node to send batches to parent chain:

Reviewer: ```suggestion
Enable your node to send batches to the parent chain:
```

```shell
--node.batch-poster.enable=true
--node.batch-poster.max-size=90000
--node.batch-poster.parent-chain-wallet.private-key=<Your Parent Chain Wallet Private Key>
```

#### 4. Disable transaction forwarding

> Disable your sequencer's forwarding transaction as the node will queue the transaction directly:

Reviewer: ```suggestion
Disable your sequencer's forwarding transactions, as the node will queue the transaction directly:
```

```shell
--execution.forwarding-target=""
```

> #### 5. Enable feed out queued transaction

Reviewer: ```suggestion
#### 5. Enable feed-out queued transactions
```

> Enable your node to feed out transaction so fullnode can receive queued transaction:

Reviewer: ```suggestion
Enable your node to feed out transactions so full node can receive queued transactions:
```

```shell
--node.feed.output.enable=true
--node.feed.output.addr=<Sequencer feed url>
--node.feed.output.port=<Sequencer feed port>
```

#### 5. Connect the node to data availability servers

> (This step only required in Anytrust mode)

Reviewer: ```suggestion
(This step is only required in Anytrust mode)
```

> (This step is only required in Anytrust mode)

Reviewer: ```suggestion
Note: This step is only required in Anytrust mode.
```

> Enable your node can send batch to DAS and get DACerts from them.

Reviewer: ```suggestion
Enable your node to send batches to DAS and get DACerts from them.
```

  ```

>   - Ensure that `/some/local/dir/arbitrum` already exists otherwise the directory might be created with `root` as owner, and the Docker container won't be able to write to it

Reviewer: ```suggestion
  - Ensure that `/some/local/dir/arbitrum` already exists otherwise, the directory might be created with `root` as owner, and the Docker container won't be able to write to it.
```

>   - Ensure that `/some/local/dir/arbitrum` already exists otherwise, the directory might be created with `root` as owner, and the Docker container won't be able to write to it.

Reviewer: ```suggestion
  - Ensure that `/some/local/dir/arbitrum` already exists; otherwise, the directory might be created with `root` as owner, and the Docker container won't be able to write to it.
```

        "rest-aggregator": {
          "enable": true,
          "urls": ["http://das-server:9877"]
        },
        "rpc-aggregator": {
          "enable": true,
          "assumed-honest": 1,
          "backends": "[{\"url\":\"http://das-server:9876\",\"pubkey\":\"YAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==\",\"signermask\":1}]"
        }
      }
    },
    "execution": {
      "forwarding-target": "",
      "sequencer": {
        "enable": true,
        "max-tx-data-size": 85000,
        "max-block-speed": "250ms"
      }
    }
  }
  ```

### Note on permissions

- The Docker image is configured to run as non-root `UID 1000`. If you are running Linux or macOS and you are getting permission errors when trying to run the Docker image, run this command to allow all users to update the persistent folders:

>   ```shell
>   mkdir /data/arbitrum
>   chmod -fR 777 /data/arbitrum
>   ```

Reviewer: ```suggestion

```shell
mkdir /data/arbitrum
chmod -fR 777 /data/arbitrum
```

```

## Optional parameters

> We show here a list of the parameters that are most commonly used when running your Orbit sequencer node. You can also use the flag `--help` for a full comprehensive list of the available parameters.

Reviewer: ```suggestion
Here's a list of the parameters that are most commonly used when running your Orbit sequencer node. You can also use the flag `--help` for a comprehensive list of available parameters.
```
